phare/data/wrangler.py

```diff
@@ -1,15 +1,30 @@
-class DataWrangler:
-    is_primal = {"bx": True, "by": False, "bz": False, "ex": False, "ey": True, "ez": True}
+primal_datasets = {
```

Comment:
this depends on the layout type. 
we'd rather use the gridlayout.py it's made for that

Reply:
can something like this live on GridLayout.py?

what's there isn't so convenient.

Reply:
please PR what you think is convenient there :)
the only think that matters is that these specific values are only true for the Yee layout.

Reply:
resolved by https://github.com/PHAREHUB/PHARE/pull/180